### PR TITLE
1.0.1 doc changes

### PR DESCRIPTION
**Pull request by cs-abhishek-shukla**

### CODE REVIEW

File: docs/contents.md [hunk 1]
```diff
@@ -100,16 +100,6 @@
...
-|:------------------------------------------------------------------------|
-
-| Playbook Name                                 | Description                                                                  |
-|:----------------------------------------------|:-----------------------------------------------------------------------------|
-| Link ATT&CK technique to Alert                | Links MITRE technique or sub-technique to Alert, based on MITRE Attack ID    |
-| Link ATT&CK technique to Alert (On Update)    | Links MITRE technique or sub-technique to Alert, based on MITRE Attack ID    |
-| Link ATT&CK technique to Incident             | Links MITRE technique or sub-technique to Incident, based on MITRE Attack ID |
-| Link ATT&CK technique to Incident (On Update) | Links MITRE technique or sub-technique to Incident, based on MITRE Attack ID |
-
 | 02 - Use Case - MITRE ATT&CK™ - Modulars |
```
Comment (cs-suyog-jinturkar) [inline docs/contents.md]: It should be TM or R?

Comment (cs-suyog-jinturkar) [reply docs/contents.md]: Please change every where..

**Review by cs-suyog-jinturkar — COMMENTED** [review]
Added review comments

File: docs/contents.md (hunk 1 above)
Comment (cs-abhishek-shukla) [reply docs/contents.md]: Fixed

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**